File: Service_eHospital_DongNai_A/Implement/me_AppListService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class me_AppListService : BaseService<me_AppList, Ime_AppListRepository>
    , Ime_AppListService
    {
    private readonly Ime_AppListRepository _me_AppListRepository;
    public me_AppListService(Ime_AppListRepository me_AppListRepository) : base(me_AppListRepository)
    {
    _me_AppListRepository = me_AppListRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/me_AppList.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { me_AppList } from './me_AppList.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class me_AppListService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'APP_id', 'APP_Name'];

    List: me_AppList[] | undefined;
    ListFilter: me_AppList[] | undefined;
    FormData!: me_AppList;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "me_AppList";
    }
}

